net/mwan3/files/usr/sbin/mwan3track

```diff
@@ -141,7 +141,7 @@ main() {
 						# https://bugs.openwrt.org/index.php?do=details&task_id=2897
 						# so get the IP address of the interface and use that instead
 						if echo $track_ip | grep -q ':'; then
-							ADDR=$(ip -6 addr ls dev "$DEVICE" scope global | sed -ne '/\/128/d' -e 's/ *inet6 \([^ \/]*\).* scope global.*/\1/p')
+							ADDR=$(ip -6 addr ls dev "$DEVICE" scope global | sed -ne '/\/128/d' -e 's/ *inet6 \([^ \/]*\).* scope global.*/\1/p' | head -n 1)
```

Comment:
This can be done without having to invoke another tool:
```suggestion
							ADDR=$(ip -6 addr ls dev "$DEVICE" scope global | sed -ne '/\/128/d' -e 's/ *inet6 \([^ \/]*\).* scope global.*/\1/{p;q}')
```

Reply:
Nice. I think it is worth having the ADDR value limited to one result. There could be the scenario of having multiple global IPv6 addresses on a single interface and without limiting it to one, it's possible that ADDR could have more than one address value passed to it and then it would break mwan3track.

Reply:
Selecting the first ipv6 global address using head -1 made mwan3track work in my case with two WAN + WWAN  ipv6 interfaces  that were returning multiple ipv6 global addresses.
I use it in the same way as the author for 4G fallback - no load balancing - WIthout it ipv6 interfaces were marked always non working.

Reply:
@ngtech I also do the same thing, the change here would still limit the IPv6 address returned to one (first match), just without having to pipe another tool, in this case `head`. It is just more optimised.

Reply:
@ngtech (and @jamesmacwhite) No argument on selecting only the first address, but see my suggestion above on how to do it without invoking `| head`.

I just updated the suggestion as I discovered that `sed`'s `s///` doesn't like multiple actions (`{p;q}`).

It would be useful if you tried the suggestion above and confirmed it works for you.

Reply:
@brianjmurrell I can confirm the suggested sed changes selects the first global address and is limited to one result, so should be OK to merge.